ee/modules/110-istio/hooks/ensure_crds_istio.go

```diff
@@ -52,5 +53,10 @@ func ensureCRDs(input *go_hook.HookInput, dc dependency.Container) error {
 	sort.Sort(semver.Collection(semvers))
 
 	CRDversionToInstall := fmt.Sprintf("%d.%d", semvers[len(semvers)-1].Major(), semvers[len(semvers)-1].Minor())
-	return ensure_crds.EnsureCRDsHandler("/deckhouse/modules/110-istio/crds/istio/"+CRDversionToInstall+"/*.yaml")(input, dc)
+
+	editionPrefix := "/deckhouse/"
```

Comment:
just prefix pls.

Reply:
Renamed in https://github.com/deckhouse/deckhouse/pull/3740/commits/37d608b4cb0d9143d5c3eede47e2f2af552a2433.